libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt

```diff
@@ -158,11 +163,14 @@ internal class InfoPanelCoordinator(
     private fun updateGuidelinePosition(
         systemBarsInsets: Insets = context.systemBarsInsets.value,
         infoPanelTop: Int = infoPanel.top,
+        maxPosPercent: Float = context.styles.infoPanelGuidelineMaxPosPercent.value
     ) {
         val parentHeight = (infoPanel.parent as ViewGroup).height
-        val maxPos = (parentHeight * GUIDELINE_MAX_POSITION_PERCENT).toInt()
-        val pos = parentHeight - infoPanelTop - systemBarsInsets.bottom
-        guidelineBottom.setGuidelineEnd(pos.coerceIn(0, maxPos))
+        val maxPos = (parentHeight * maxPosPercent).toInt() - systemBarsInsets.bottom
```

Comment:
You didn't subtract `systemBarsInsets.bottom` for maxPos before. Is this behaviour change intentional?

Reply:
yes